pom.xml

```diff
@@ -55,6 +55,11 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.htmlparser.jericho</groupId>
+      <artifactId>jericho-html</artifactId>
+      <version>3.1</version>
+    </dependency>
```

Comment:
What is Jericho used for?

Reply:
It's an HTML parser so it allows us to ignore all the tags in the comments (like <p>, <br>, <b>, etc.)